types/jest-image-snapshot/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for jest-image-snapshot 3.1
+// Type definitions for jest-image-snapshot 4.1.0
```

Comment:
```suggestion
// Type definitions for jest-image-snapshot 4.1
```
this would still be only `major.minor` ver